src/components/Reports/SavingsFundTransfer/TransfersPage/TransfersPage.test.tsx

```diff
@@ -0,0 +1,322 @@
+
+    await waitFor(() => {
+      const modal = getByRole('dialog');
+      expect(modal).toBeInTheDocument();
+
+      // Check modal contains form elements
+      expect(
+        getByRole('combobox', { name: /from account/i }),
+      ).toBeInTheDocument();
+      expect(
+        getByRole('combobox', { name: /to account/i }),
+      ).toBeInTheDocument();
+      expect(getByRole('radio', { name: /one time/i })).toBeInTheDocument();
+    });
+  });
+
+  it('should render with custom title', () => {
+    const customTitle = 'Custom Transfer Page Title';
+    const { getByText } = render(<Components title={customTitle} />);
+
+    expect(getByText(customTitle)).toBeInTheDocument();
+  });
+
+  it('should handle fund data correctly', () => {
+    const { getAllByText } = render(<Components />);
+
+    // Verify all fund types are handled
+    const staffAccountFund = mockData.funds.find(
+      (f) => f.type === StaffSavingFund.StaffAccount,
+    );
+    const staffSavingsFund = mockData.funds.find(
+      (f) => f.type === StaffSavingFund.StaffSavings,
+    );
+    const staffConferenceFund = mockData.funds.find(
+      (f) => f.type === StaffSavingFund.StaffConferenceSavings,
+    );
+
+    expect(staffAccountFund).toBeDefined();
+    expect(staffSavingsFund).toBeDefined();
+    expect(staffConferenceFund).toBeDefined();
+
+    // Check that balances are displayed using getAllByText for amounts that appear multiple times
+    expect(getAllByText('$15,000.00').length).toBeGreaterThan(0); // Staff Account balance from mock
+    expect(getAllByText('$2,500.00').length).toBeGreaterThan(0); // Staff Savings balance from mock
+    expect(getAllByText('$500.00').length).toBeGreaterThan(0); // Staff Conference Savings balance from mock
+  });
+
+  it('should maintain responsive layout', () => {
+    const { container } = render(<Components />);
+
+    // Check that responsive classes or styles are applied - look for MUI Box components
+    const boxContainers = container.querySelectorAll('.MuiBox-root');
+    expect(boxContainers.length).toBeGreaterThan(0);
+
+    // Check for Container component which handles responsive layout
+    const containerElements = container.querySelectorAll('.MuiContainer-root');
+    expect(containerElements.length).toBeGreaterThan(0);
+  });
+
+  it('should pass correct props to transfer modal', async () => {
+    const { getByRole, container } = render(<Components />);
+
+    // Open modal
+    const transferButtons = container.querySelectorAll(
+      '[data-testid="OutboxIcon"]',
+    );
+    const firstTransferButton = transferButtons[0].closest('button');
+    userEvent.click(firstTransferButton!);
+
+    await waitFor(() => {
+      expect(getByRole('dialog')).toBeInTheDocument();
+
+      // Verify modal has the expected form structure
+      expect(getByRole('button', { name: /submit/i })).toBeInTheDocument();
+      expect(getByRole('button', { name: /cancel/i })).toBeInTheDocument();
+    });
+  });
```

Comment:
If we change the test above this one called `should display transfer modal with correct type` to check the parameters passed through are being used correctly, we can remove this one.